draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -848,6 +848,152 @@ Content-Type: text/plain
+</t>
+<t>
+   Two origins are distinct if they differ in scheme, host, or port. Even
+   when it can be verified that the same authority controls two origins that
+   might differ only by scheme name, the two namespaces under those origins
+   are distinct unless explicitly aliased by that authority.
+</t>
+<t>
+   Note that the presence of a URI does not imply that there is always an
+   HTTP server at the identified origin listening for connections on
+   the identified host and port. Anyone can mint a URI. What the origin
+   determines is who has the right to respond authoritatively to requests that
+   target the identified resource. The delegated nature of registered names
+   and IP addresses creates a federated namespace whether or not an HTTP
+   server is present.
+</t>
+<t>
+   Although HTTP is independent of the transport protocol, the "http" scheme
+   is specific to associating authority with whomever controls the origin
+   server listening for TCP connections on the indicated port of whatever
+   host is identified within the authority component. This is a very weak
+   sense of authority because it depends on both client-specific name
+   resolution mechanisms and communication that might not be secured from
+   man-in-the-middle attacks. Nevertheless, it is a sufficient minimum for
+   binding "http" identifiers to an origin server for consistent resolution
+   within a trusted environment.
+</t>
+<t>
+   If the host identifier is provided as an IP address, the origin server is
+   the listener (if any) on the indicated TCP port at that IP address.
+   If host is a registered name, the registered name is an indirect identifier
+   for use with a name resolution service, such as DNS, to find an address for
+   an appropriate origin server.
+</t>
+<t>
+   Access to "http" services is not limited to the authority mechanism.
+   For example, the Alt-Svc header field <xref target="RFC7838"/> allows an
+   origin server to explicitly alias its own namespace to an alternative
+   service at a different origin. Access to "http" identified resources might
+   also be provided by other protocols outside the scope of this document.
+</t>
+<t>
+   The "https" scheme, in contrast, associates authority based on the ability
+   of a server to use the private key associated with a certificate that
+   the client considers to be trustworthy for the identified host.
+   If a server presents a certificate that verifiably applies to the host,
+   along with proof that it controls the private key, then a client will
+   accept a secured connection to that server as being authoritative for all
+   origins with the same host. However, each origin remains distinct (from
+   origins with a different scheme or port) unless explicitly aliased by an
+   origin server verified to be under the same authority.
+</t>
+<t>
+   What the client is relying on for "https" authority is that there exists
```

Comment:
```suggestion
   Authority in the "https" scheme relies on
```

Reply:
Tweaked slightly, but I am deliberately trying to describe the client mechanism here, whereas the other paragraphs covered a neutral perspective.